src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,33 @@
+package step2;
+
+public class Calculator {
+
+  private final static String SPLIT_SPACE = " ";
+
+  public int calculate(String input) {
+
+    if(isEmpty(input)) {
```

Comment:
```suggestion
    valid(input)
```
유효성 검사하는 메서드로 분리해보면 어떨까요?